apiserver/facades/controller/caasunitprovisioner/provisioner_test.go

```diff
@@ -267,57 +267,8 @@ func (s *CAASProvisionerSuite) TestUpdateApplicationsUnitsNoTags(c *gc.C) {
 	})
 }
 
-func (s *CAASProvisionerSuite) TestUpdateApplicationsUnitsWithTags(c *gc.C) {
```

Comment:
I am not sure why this test is no longer needed? There could be several reasons for why the units are tagged, not necessarily just for managing them by juju but could be for reporting/other reasons...

In other words, I think that juju-managed units and tagged units are 2 separate concerns. I think we still want to have this test.

Reply:
sorry, @anastasiamac , `updateJujuManagedUnitsFromCloud` is the process inside `updateJujuManagedUnitsFromCloud` which has currently been ignored and we always use `updateUnitsFromCloud` instead. `s.st.application.jujuManagedUnits = true` is the toggle for this test and it's currently always `false`.

Reply:
Ah, makes sense :) Thank you for the explanation!

Reply:
np :)  it's not easy to find it without digging into the funcs called here.